service/schema.py

```diff
@@ -0,0 +1,25 @@
+import json
+from jsonschema import validate
+
+schema = {
+    "type": "object",
+    "properties": {
+        "item_one": {"type": "string"},
+        "item_two": {"type": "number"}
+    }
+}
+
+
+def validate_schema(message_payload):
+    """
+    Validate message queue payload against json schema
+        - message_payload: queue item message
+    """
+    message_payload = json.dumps(message_payload)
+    try:
+        validate(instance=message_payload, schema=schema)
+        return True
```

Comment:
return True or None is quite odd, that's a code smell from where we're catching the exception (see later comment). If we raise and handle the logic will get cleaner.

Reply:
raised and handles the logic - fixed